omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -96,10 +96,10 @@
             delete "#{install_dir}/embedded/lib/*.la"
         end
 
-        if linux?
+        if linux_target?
             # Fix pip after building on extended toolchain in CentOS builder
             if redhat? && ohai["platform_version"].to_i == 6
```

Comment:
```suggestion
            if redhat_target? && ohai["platform_version"].to_i == 6
```

Reply:
This is on purpose, these commands are only to be run when building on a centos builder (which we use when building for redhat)